src/plugins/simulator/simulations/brainwaves-gripper/brainwaves-gripper.tsx

```diff
@@ -160,9 +161,13 @@ function step({ frame, variables }: ISimulationProps) {
     const getPressureValue = () => {
       if (!gripperValue) return 0;
       if (modeVariable?.currentValue === kSimulationModeTemperature) {
-        return gripperValue > minTemperatureValue ? (gripperValue - minTemperatureValue) * 200 : 0;
+        return gripperValue > minTemperatureValue
```

Comment:
These formulas could use an explanatory comment.
I have no idea what the result of this combination of the gripper value, minTemperature, maxPressure, and some other constants actually is going to be or if it is correct...
A Jest test for this rather complex method also wouldn't hurt.

Reply:
@bgoldowsky I added comments and tweaked variable names, so I hope it's more clear what the intention of this function is.

Your suggestion of a jest test and general confusion here is making me realize that these simulations have grown up a bit and probably deserve some refactoring attention. In addition to jest tests, I think it would be worth splitting up these files into separate files for constants/variable definitions, the step functions, and components. I'm not sure if it makes sense to do that for this story, which is pretty limited in scope, though.